rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -95,7 +104,9 @@ ifdef::openshift-rosa[]
 
 * **{hcp-title} multi-architecture cluster update.** {hcp-title-first} clusters created before 25 July, 2024 will migrate to a multi-architecture image on their next upgrade allowing you to use {AWS} Arm-based Graviton instance types for your workloads. For more information, see xref:../upgrading/rosa-hcp-upgrading.adoc#rosa-upgrade-options_rosa-hcp-upgrading[Upgrading ROSA with HCP clusters].
 
-* **{hcp-title} cluster node limit update.** {hcp-title} clusters can now scale to 500 worker nodes. This is an increase from the previous limit of 250 nodes. The 250 node limit is an increase from the previous limit 90 nodes on 26 August, 2024. For more information, see xref:../rosa_planning/rosa-hcp-limits-scalability.adoc#tested-cluster-maximums-hcp-sd_rosa-hcp-limits-scalability[ROSA with HCP cluster maximums].
+* **{hcp-title} cluster node limit update.** {hcp-title} clusters can now scale to 500 worker nodes. This is an increase from the previous limit of 250 nodes. The 250 node limit is an increase from the previous limit 90 nodes on 26 August, 2024.
```

Comment:
Worker nodes > compute nodes.

https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html-single/architecture/index#openshift-architecture-common-terms_architecture-overview

![Screenshot From 2025-03-19 14-26-22](https://github.com/user-attachments/assets/2dae9800-f78a-4033-b14b-adac31dbcef9)

Reply:
Not sure if this is a statement that worker node is preferred over / greater than compute node or a direction to change from worker node to compute node; there are 500+ instances of worker and 1000+ instances of compute in the openshift docs repo